Automattic/WP-Job-Manager
# Feature: Promoted Jobs

Promoted Jobs feature branch.

## Changes proposed in this Pull Request

- It introduces the Promoted Jobs feature to the plugin, where users can promote a job through the JobTarget partner.

## Testing instructions

- This version was already tested pointing to staging and production.
  - If you have the env configured, you can also test pointing to a development wpjobmanager.com site.
- Make sure the Job Listing displays the Promote button and the plugin is not broken.


----

| Plugin build for 8bd16f769ed918328e0b2fc4755e1228ca26e132 <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/10/wp-job-manager-zip-2451-8bd16f76.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/10/2451-8bd16f76)             |
